cypress/e2e/file-menu/delete.cy.ts

```diff
@@ -0,0 +1,73 @@
+import {
+    // file-menu
+    deleteVisualization,
+    saveVisualizationAs,
+    openVisByName,
+    // table
+    expectVisTitleToEqual,
+    expectTableToBeVisible,
+    expectStartScreenToBeVisible,
+} from '../../helpers/index'
+
+const TEST_VIS_TITLE = `delete-test-${new Date()
+    .toISOString()
+    .replace(/[:.]/g, '-')}`
+
+const createTestVisualization = (title) => {
+    openVisByName('Inpatient: Cases last quarter (case)')
+
+    // capture the current visualization id from the hash route BEFORE saving
+    cy.url().then((url) => {
+        const match = url.match(/#\/([A-Za-z0-9]+)$/)
+        cy.wrap(match[1]).as('initialVisId')
+    })
+
+    // save as a new visualization for the test
+    saveVisualizationAs(title)
+    // after saving as a new visualization the id should have changed
+    cy.get('@initialVisId').then((initialVisId) => {
+        cy.url().should('not.match', new RegExp(`${initialVisId}$`))
+    })
```

Comment:
Instead of using `cy.url()` you could use `cy.location('pathname)` it should be possible to make this a bit simpler.